jest.config.js

```diff
@@ -2,174 +2,52 @@
 // https://jestjs.io/docs/en/configuration.html
 
 module.exports = {
-  // All imported modules in your tests should be mocked automatically
-  // automock: false,
-
-  // Stop running tests after the first failure
-  // bail: false,
-
-  // Respect "browser" field in package.json when resolving modules
-  // browser: false,
-
-  // The directory where Jest should store its cached dependency information
-  // cacheDirectory: "/var/folders/kq/vr8smgcj4r31rf9lzznt_l280000gn/T/jest_dx",
-
   // Automatically clear mock calls and instances between every test
-  // clearMocks: true,
+  clearMocks: true,
 
   // Indicates whether the coverage information should be collected while executing the test
-  // collectCoverage: false,
-
-  // An array of glob patterns indicating a set of files for which coverage information should be collected
-  // collectCoverageFrom: null,
+  collectCoverage: true,
 
   // The directory where Jest should output its coverage files
   coverageDirectory: 'coverage',
 
-  // An array of regexp pattern strings used to skip coverage collection
-  // coveragePathIgnorePatterns: [
-  //   "/node_modules/"
-  // ],
-
-  // A list of reporter names that Jest uses when writing coverage reports
-  // coverageReporters: [
-  //   "json",
-  //   "text",
-  //   "lcov",
-  //   "clover"
-  // ],
-
-  // An object that configures minimum threshold enforcement for coverage results
-  // coverageThreshold: null,
-
-  // Make calling deprecated APIs throw helpful error messages
-  // errorOnDeprecated: false,
-
-  // Force coverage collection from ignored files usin a array of glob patterns
-  // forceCoverageMatch: [],
-
-  // A path to a module which exports an async function that is triggered once before all test suites
-  // globalSetup: null,
-
-  // A path to a module which exports an async function that is triggered once after all test suites
-  // globalTeardown: null,
-
-  // A set of global variables that need to be available in all test environments
-  // globals: {},
-
   // An array of directory names to be searched recursively up from the requiring module's location
-  // moduleDirectories: ['node_modules', 'src'],
+  moduleDirectories: [
+    "node_modules",
+    "<rootDir>/src"
+  ],
 
   // An array of file extensions your modules use
-  // moduleFileExtensions: [
-  //   "js",
-  //   "json",
-  //   "jsx",
-  //   "node"
-  // ],
+  moduleFileExtensions: [
+    "ts",
+    "tsx",
+    "js"
```

Comment:
Do we need js here?

Reply:
Was thinking it would be good for interoperability, in cases where users have existing js modules they want to copypasta to further kick start their efforts, but I'm not dead set on this. If we think it could cause trouble or be misleading/confusing - I'm happy to remove it. @ibolton336 @dgutride thoughts on this?

Reply:
Oops, just realized this was for the jest config. It can probably go away, lemme verify real quick

Reply:
Looks like it's required by jest

<img width="367" alt="screen shot 2019-03-06 at 1 15 49 pm" src="https://user-images.githubusercontent.com/5942899/53903575-08a30080-4012-11e9-94db-c3a66464e7ef.png">